tests/integration/components/frost-chart-svg-plot-stacked-bar-test.js

```diff
@@ -0,0 +1,109 @@
+import {expect} from 'chai'
+import {$hook} from 'ember-hook'
+import {integration} from 'ember-test-utils/test-support/setup-component-test'
+import hbs from 'htmlbars-inline-precompile'
+import {beforeEach, describe, it} from 'mocha'
+
+const test = integration('frost-chart-svg-plot-stacked-bar')
+
+describe(test.label, function () {
+  test.setup()
+
+  const data = [
+    {category: 'Thing One', a: 1, b: 2, c: 3},
+    {category: 'Thing Two', a: 3, b: 2, c: 1}
+  ]
+  const xDomain = ['Thing One', 'Thing Two']
+  const yDomain = [0, 6]
+
+  beforeEach(function () {
+    this.setProperties({
+      data,
+      xDomain,
+      yDomain
+    })
+
+    this.render(hbs`
+      <style>
```

Comment:
Is this styling necessary for the test?

Reply:
Without it the SVG element won't have a proper height and will throw errors in the console, though the tests will still pass. Including the style removes the console errors and renders a real looking chart for the integration tests.

Reply:
Do we need to point this styling requirement out in the dummy app so users know to apply (appropriate) height to their chart?

Reply:
Good idea, I added a note to the dummy app.